lib/dor/services/client/mutate.rb

```diff
@@ -26,7 +26,7 @@ def apply_admin_policy_defaults
         end
 
         # Updates the object
-        # @param [Cocina::Models::RequestDRO,Cocina::Models::RequestCollection,Cocina::Models::RequestAPO] params model object
+        # @param [Cocina::Models::DRO,Cocina::Models::Collection,Cocina::Models::APO] params model object
```

Comment:
It turns out `RequestAPO` wasn't a real thing, and neither is `Cocina::Models::APO`. Can you change to `Cocina::Models::AdminPolicy`?

Reply:
good catch; done.